test/pools/integration/Cases.t.sol

```diff
@@ -359,4 +363,43 @@ contract TestCases is PoolsDeployer, Test {
         (uint128 totalIssuance2,) = multiShareClass.metrics(scId);
         assertEq(totalIssuance2, 55);
     }
+
+    function testNotifySharePrice() public {
+        (PoolId poolId, ShareClassId scId) = testPoolCreation();
+        D18 sharePrice = d18(100, 1);
+        D18 poolPerEUR = d18(1, 3); // NOTE: 3 EUR = 1 USD
+        D18 poolPerUSDC_C2 = d18(1, 1); // NOTE: 1 USDC_C2 = 1 USD
+
+        (bytes[] memory cs, uint256 c) = (new bytes[](5), 0);
+        cs[c++] = abi.encodeWithSelector(poolRouter.setTransientPrice.selector, EUR.addr(), poolPerEUR);
```

Comment:
@mustermeiszer This test is broken and I cannot assume it ever worked because `hub.notifyAssetPrice` does not use the transient valuation but the valuation from the holdings. Did you intend to conditionally allow the transient valuation for `hub.notifyAssetPrice` or should `setTransientPrice` be replaced with creating a holding and setting a price for that?